problem_builder/public/js/student_answers_dashboard.js

```diff
@@ -0,0 +1,299 @@
+function StudentAnswersDashboardBlock(runtime, element) {
+    'use strict';
+    var $element = $(element);
+
+    // Pagination
+
+    var Result = Backbone.Model.extend({
+
+        initialize: function(attrs, options) {
+            _.each(_.zip(Result.properties, options.values), function(pair) {
+                this.set(pair[0], pair[1]);
+            }, this);
+        }
+
+    }, { properties: ['section', 'subsection', 'unit', 'type', 'question', 'answer', 'username'] });
+
+    var Results = Backbone.PageableCollection.extend({
+
+        model: Result,
+
+        getCurrentPage: function(object) {
+            var currentPage = this.state.currentPage;
+            if (object) {
```

Comment:
`object` is a confusing name for this parameter - it sounds like this method is expecting an object as its parameter. How about `fetchObject` or `returnObject` or `wantsObject`?